modules/install-sno-requirements-for-installing-on-a-single-node.adoc

```diff
@@ -9,6 +9,8 @@ Installing {product-title} on a single node alleviates some of the requirements
 
 * *Administration host:* You must have a computer to prepare the ISO, to create the USB boot drive, and to monitor the installation.
 
+* *Bare metal installation:* Installing {product-title} on a single node on bare metal requires you specify the `platform.none: {}` parameter in the `install-config.yaml` configuration file.
```

Comment:
s/requires/requires that

Reply:
Thanks for your feedback Michael. I have applied your suggestion. It now reads...

Bare metal installation: Installing OpenShift Container Platform on a single node on bare metal requires that you specify the platform.none: {} parameter in the install-config.yaml configuration file.